### PR TITLE
Add sendSMS connector files: Swagger, apiProperties, and README

### PR DESCRIPTION
**Pull request by ElenaMainerici**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by ElenaMainerici** [comment]
@microsoft-github-policy-service agree company="sendSMS"

File: certified-connectors/sendSMS/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/message_send": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
```
Comment (vmanoharas) [inline certified-connectors/sendSMS/apiDefinition.swagger.json]: 'default' response should not be allowed, please correct this.

File: certified-connectors/sendSMS/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+            }
+          ]
+        }
+      }
+    },
+    "/message_status": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
```
Comment (vmanoharas) [inline certified-connectors/sendSMS/apiDefinition.swagger.json]: 'default' response should not be allowed, please correct this.

File: certified-connectors/sendSMS/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,248 @@
+{
+  "swagger": "2.0",
```
Comment (vmanoharas) [inline certified-connectors/sendSMS/apiDefinition.swagger.json:2]: Kindly update below two properties. 

The 'contact' property is required. ValuePath: /info
The 'x-ms-connector-metadata' property is required.

Please refer the template from the link below. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json

**Review by vmanoharas — COMMENTED** [review]
Hello @ElenaMainerici, 

Kindly resolve review comments.

**Comment by ElenaMainerici** [comment]
Hi @vmanoharas,

I've updated the Swagger configuration for the SendSMS connector based on your feedback. Here's what I changed:
- Updated response schemas for more accurate information.
- Improved visibility settings for parameters in the UI.
- Added necessary security details and updated metadata.

Could you take a look when you have a moment? I appreciate your help and am open to any further suggestions to make sure everything is top-notch.

Thanks!

**Comment by vmanoharas** [comment]
> Hi @vmanoharas,
> 
> I've updated the Swagger configuration for the SendSMS connector based on your feedback. Here's what I changed:
> 
> * Updated response schemas for more accurate information.
> * Improved visibility settings for parameters in the UI.
> * Added necessary security details and updated metadata.
> 
> Could you take a look when you have a moment? I appreciate your help and am open to any further suggestions to make sure everything is top-notch.
> 
> Thanks!

Thanks you very much @ElenaMainerici, will review and provide my feedback/approval.

File: certified-connectors/sendSMS/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.sendsms.ro/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.sendsms.ro/ro/prelucrarea-datelor-cu-caracter-personal-679-2016-gdpr/"
+    },
+    {
+      "propertyName": "Categories",
```
Comment (vmanoharas) [inline certified-connectors/sendSMS/apiDefinition.swagger.json:344]: The connector categories should be separated with semicolon (;) not with comma.

**Review by vmanoharas — COMMENTED** [review]
Hello @ElenaMainerici, 

Kindly fix one comment.

**Comment by ElenaMainerici** [comment]
Done :)

**Review by vmanoharas — COMMENTED** [review]
Hello @ElenaMainerici, 

Thank you very much for resolving issues. 

Your merge commit has an issue, it has not merged properly, please refer the apiProperties validation checker error. Please recommit or close this PR and open a new PR. Please don't merge it until the review is completed.

image

**Pull request closed without merging**